cmd/kubernetesDeploy.go

```diff
@@ -38,7 +39,20 @@ func runKubernetesDeploy(config kubernetesDeployOptions, telemetryData *telemetr
 	telemetryData.Custom1 = config.DeployTool
 
 	if config.DeployTool == "helm" || config.DeployTool == "helm3" {
-		return runHelmDeploy(config, utils, stdout)
+		err := runHelmDeploy(config, utils, stdout)
+		// download and execute teardown script
+		if len(config.TeardownScript) > 0 {
```

Comment:
should we run the `teardownScript` after the `setupScript` since the teardown maybe removing some resources that setup may create ?

Reply:
Teardown should run at the end to clean up things, thus everything else like `install`and `test` should be complete.

Am I overlooking something?